packages/manager/modules/pci/src/projects/project/training/dashboard/install/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,35 @@
+{
+  "pci_projects_project_training_install_section_1_title": "Installation du client",
+  "pci_projects_project_training_install_section_2_title": "Authentification",
+  "pci_projects_project_training_install_section_3_title": "Commandes",
+  "pci_projects_project_training_install_section_1_command_message_1": "Se placer dans le répertoire d'installation voulu et lancer la commande suivante vous permettra de télécharger le client ElementAI sur votre poste de travail et d'installer la configuration OVH :",
+  "pci_projects_project_training_install_section_1_command_message_2": "Aprés l'installation, l'executable ",
+  "pci_projects_project_training_install_section_1_command_message_3": " sera disponible dans le répertoire d'installation",
+  "pci_projects_project_training_install_section_1_command_message_4": "Pour finaliser votre installation ajouter cette variable d'environnement: ",
+  "pci_projects_project_training_install_section_1_command_message_5": "export EAI_PROFILE=GRA",
+
+  "pci_projects_project_training_install_section_2_message_1": "L'authentification sur la plateforme peut se faire à partir de tous les ",
```

Comment:
keep in mind that splitting a single sentence in multiple translations values is not recommended as it could break the meaning of the translated result when merging parts, is it possible that this sentence is kept as a single translation value ?

Reply:
We have a ahref link between both message. If you have an elegant way to do this, you are welcome